### DISCUSSION TITLE
Seeking Assistance with Migrating from QuickBooks Desktop to Odoo 17 Community Edition

### DISCUSSION THREAD

**Discussion by zibbonkhadka** [OCA/odoo-community.org#184] [General]
Hello,

Why is it so difficult to find Odoo partners/integrators/developers in the United States who can assist with migration? We are planning to migrate from QuickBooks Desktop to Odoo 17 Community Edition and need help with the process.

Additionally, why is it so hard to find companies/users in the United States using the Community Edition? It would be great to hear about others' experiences.

**Comment by vdewulf** [comment]
Hello,

You might contact the OCA Members in the United States. The list is on the oca website: 
https://odoo-community.org/members/country/united-states-235?

There is also the list of the corresponding companies here: 
https://odoo-community.org/integrators/country/united-states-235

OCA members are people wanting to support the non-profit association and purchasing a yearly membership at 50€. There is no certification of any kind regarding the quality of work provided by the people listed on the OCA website. It is more about shared values and support of open source or giving back to the community as they use OCA Apps and modules.

Regarding the Odoo Community Edition, I can only guess that the Odoo marketing is doing a great job at not making it clear that Community Edition can be an viable option. The market there is maybe less mature than in Europe, where we see older partners working with either both edition or even wanting to be totally independant from the proprietary licences, thus focusing on the Community Edition but those are also quite rare if you get the full picture of how many Odoo integrators exist.

I'm sure other have opinion on this topic to share!

In the meantime, I hope the list provided give you a first hint!

**Reply by zibbonkhadka** [reply]
Will do. Thank you so much.

**Comment by binhexsystems** [comment]
Hello @zibbonkhadka , my team has experience with this process. Feel free to get in touch with us, we have many clients in the US with CE.
https://binhex.cloud

**Reply by zibbonkhadka** [reply]
I have sent an email.

**Comment by rvalyi** [comment]
> Why is it so difficult to find Odoo partners/integrators/developers in the United States who can assist with migration?

Hello @zibbonkhadka you are kind of discovering what I call the "Odoo Fermi Paradox".

And well an hypothesis for the solution to the famous Fermi Paradox is the hypothesis of a "super predator civilization"
https://medium.com/@theRLtoddspring/on-the-most-terrifying-possibility-of-the-fermi-paradox-33195068a047

The Sci-Fi speculations usually don't envolve a kiss of death via some magic partnership but this is how it usually happens in the Odoo galaxy however...

An other good Sci-Fi reference to understand the Odoo world is the Time Machine 1960 movie from the H. G. Well novel. You should just ask yourself why are all these Odoo partners like the Elois, always young, happy and unaware of the OCA? What would happen to them later? 1984 is a good one too.

In a recent viral post, the ERPNext CEO claimed Odoo was a VC inflated startup about to burst:
https://frappe.io/blog/erpnext/odoos-marketing-blitz-in-india

He was grossly wrong about the amount of money raised by Odoo SA. Still he was right about something: Odoo SA has lot of cash and this money do come from somewhere. Well I think I gave you enough clues...

Good luck and be careful with the mass of non serious integrator, ideally double check their claimed OCA references. Also remember about the "market for lemon" theory.
